polkadot/runtime/parachains/src/coretime/mod.rs

```diff
@@ -83,6 +84,8 @@ enum CoretimeCalls {
 	SetLease(pallet_broker::TaskId, pallet_broker::Timeslice),
 	#[codec(index = 19)]
 	NotifyCoreCount(u16),
+	#[codec(index = 20)]
+	SwapLeases(ParaId, ParaId),
```

Comment:
Given that this is a temporary call that will be removed once no longer needed, it might make sense to use index 99 or something.

Reply:
Done